src/main/kotlin/no/nav/familie/ks/sak/integrasjon/sanity/domene/SanityBegrunnelse.kt

```diff
@@ -1,84 +1,84 @@
 package no.nav.familie.ks.sak.integrasjon.sanity.domene
 
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.UtdypendeVilkårsvurdering
 import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Vilkår
-import no.nav.familie.ks.sak.kjerne.beregning.domene.Årsak
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.VilkårResultat
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.Person
 import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
+import java.math.BigDecimal
 
 data class SanityBegrunnelse(
     val apiNavn: String?,
     val navnISystem: String,
-    val vilkaar: List<Vilkår>? = null,
-    val rolle: List<PersonType> = emptyList(),
-    val lovligOppholdTriggere: List<VilkårTrigger>? = null,
-    val bosattIRiketTriggere: List<VilkårTrigger>? = null,
-    val borMedSokerTriggere: List<VilkårTrigger>? = null,
-    val ovrigeTriggere: List<ØvrigTrigger>? = null,
-    val endringsaarsaker: List<Årsak>? = null,
-    val hjemler: List<String> = emptyList(),
-    val hjemlerFolketrygdloven: List<String> = emptyList(),
-    val endretUtbetalingsperiodeTriggere: List<EndretUtbetalingsperiodeTrigger>? = null,
-    val endretUtbetalingsperiodeDeltBostedUtbetalingTrigger: EndretUtbetalingsperiodeDeltBostedTriggere? = null
+    val type: SanityBegrunnelseType,
+    val vilkår: List<Vilkår>,
+    val rolle: List<PersonType>,
+    val utdypendeVilkårsvurderinger: List<UtdypendeVilkårsvurdering>,
+    val triggere: List<Trigger>,
+    val hjemler: List<String>
 )
 
+enum class SanityBegrunnelseType {
+    STANDARD,
+    TILLEGGSTEKST,
+    ENDRINGSPERIODE
+}
+
 data class SanityBegrunnelserResponsDto(
     val ms: Int,
     val query: String,
     val result: List<SanityBegrunnelseDto>,
     val endringsaarsaker: List<String>? = emptyList()
 )
 
+enum class Trigger {
+    SATSENDRING,
+    BARN_DØD,
+    DELTID;
```

Comment:
Hmm, kanskje det er like greit å kalle dette for DELTID_BARNEHAGEPLASS eller GRADERT_BARNEHAGEPLASS?
Bare slik at det kommer litt mer klart fram.